src/components/NewCommentForm.tsx

```diff
@@ -1,13 +1,17 @@
 import classNames from 'classnames';
 import React, { useState } from 'react';
 import { CommentData } from '../types/Comment';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
+import { actions as newCommentActions } from '../features/newComment/newComment'; // eslint-disable-line max-len
 
 type Props = {
   onSubmit: (data: CommentData) => Promise<void>;
 };
 
 export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
-  const [submitting, setSubmitting] = useState(false);
+  // const [submitting, setSubmitting] = useState(false);
```

Comment:
The `submitting` state is commented out. If you intend to use this state to manage form submission loading, consider uncommenting and integrating it with the form logic.